src/components/organisms/PageHeader/ClusterSelection.tsx

```diff
@@ -162,9 +160,8 @@ const ClusterSelection = ({previewResource}: {previewResource?: K8sResource}) =>
           {isKubeConfigPathValid ? (
             <>
               <S.Divider type="vertical" />
-              <Tooltip mouseEnterDelay={TOOLTIP_DELAY} title={ClusterModeTooltip} placement="right">
+              <Tooltip mouseEnterDelay={TOOLTIP_DELAY} mouseLeaveDelay={0} title={ClusterModeTooltip} placement="right">
                 <S.Button
-                  disabled={Boolean(previewType === 'cluster' && previewLoader.isLoading) || isClusterActionDisabled}
```

Comment:
I think sometimes the button has to be disabled?

Reply:
Having the button disabled caused this problem, even if the button is enabled it will have loading instead of text and the onClick function has the same check